README.md

```diff
@@ -91,3 +91,5 @@ recorder:
 | rotate | number | 0 | Value to rotate the map by (default is in deg, but a value like `2rad` is valid too)
 | crop | Object | {top: 0, bottom: 0, left: 0, right: 0} | Crop the map
 | min_height | string | 0 | The minimum height of the card the map is displayed in, regardless of the map's size itself. Suffix with 'w' if you want it to be times the width (ex: 0.5625w is equivalent to a picture card's 16x9 aspect_ratio)
+
+Colors can be any hex code in the card config, e.g. `path_color = '#FF0000'`.
```

Comment:
They can actually be any valid CSS value, so also `red` or `rgb(255,255,255)` or `rgba(255,255,255,0.8)`.

Reply:
Extended that as well.